docker/README.md

```diff
@@ -40,10 +40,10 @@ services:
             - mongodb
```

Comment:
As additional comment regarding to documentation, I'd suggest the follwoing one in docs/administrationGuide.md. Change:

```
-   **port**: UDP port where the IoT Agent will be listening. E.g.: 60001.
```

to

```
-   **port**: UDP port where the IoT Agent will be listening. Note that default CoAP port according to [its specification](https://tools.ietf.org/html/rfc7252) is 5683.
```

Reply:
fixed by 8e44d03b79818ded53ed855771f0d981180dcfd5

Reply:
fuxed by 8e44d03b79818ded53ed855771f0d981180dcfd5

Reply:
no code added but tests start to fail? :thinking:

Reply:
Weird... [the changes in Orion in the last days](https://github.com/telefonicaid/fiware-orion/commits/master) should have impacted IOTAs.

Another possible cause is a modification in the ioagent-node-lib merged this morning (PRhttps://github.com/telefonicaid/iotagent-node-lib/pull/819 ). I was supposed to be a bugfix, but maybe something has broken. To be investigated...

Reply:
I have done some investigation on this. It seems is not a problem of ContextBroker or iotagent-node-lib.

With the code at current HEAD in master (commit 18d6ebc5933f8c098e58ca0a499c24aa2951eb5c) I get the following result:

```
 35 passing (38s)
  11 pending
  10 failing
```

This component depends on two libraries: iotagent-node-lib and lwm2m-node-lib in master. Both have recently received a recent PR merge in master: https://github.com/telefonicaid/iotagent-node-lib/pull/819 and https://github.com/telefonicaid/lwm2m-node-lib/pull/152.

I started rolling back both (moving the dependency from master HEAD to a previous commit, just the one before merging in each case, see: https://github.com/telefonicaid/iotagent-node-lib/commits/master and https://github.com/telefonicaid/lwm2m-node-lib/commits/master) and running again the tests. And they worked.

```
$ git diff
diff --git a/package.json b/package.json
index aa2e6de..22fa352 100644
--- a/package.json
+++ b/package.json
@@ -50,8 +50,8 @@
     "cheerio": "1.0.0-rc.2",
     "xmldom": "0.1.27",
     "logops": "2.1.0",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#09f3bdbebbdd8e7f4112f14d65f55b6fcc8b5116",
+    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#955e14590f8e37ccf829646ad8e453262a724ec4"
   },

$ npm run tests
...
  45 passing (38s)
  11 pending
```

Next step: move iotagent-node-lib to master and check again. Again, tests ok.

```
$ git diff
diff --git a/package.json b/package.json
index aa2e6de..65f20d8 100644
--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
     "xmldom": "0.1.27",
     "logops": "2.1.0",
     "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
-    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#master"
+    "lwm2m-node-lib": "git://github.com/telefonicaid/lwm2m-node-lib.git#955e14590f8e37ccf829646ad8e453262a724ec4"
   },
   "devDependencies": {
     "coveralls": "~3.0.2",

$ npm run test
  45 passing (39s)
  11 pending
```

Thus, final step: moving lwm2m-node-lib to master and check again, now getting (as expected) the fail:

```
$ npm test

 35 passing (38s)
  11 pending
  10 failing
```

**Conclusion:** modifications introduced by PR https://github.com/telefonicaid/lwm2m-node-lib/pull/152 authored by @versatildefuy seems to be the cause of the broken unit tets.

Two options:

1. Revert PR https://github.com/telefonicaid/lwm2m-node-lib/pull/152 and launch a new one hopefully without the problem.
2. Do an extra PR on https://github.com/telefonicaid/lwm2m-node-lib to solve the problem.

It depends on how it could take to find a fix. If the problem is unknown and it will take time to analyze it and solve it, let's go for option 1. If the problem is somehow clear and the solution is easy to implement, let's go for option 2.

As far as I see in your github profile both @versatildefuy and you work for the same company. Maybe you can discuss it internally and tell us what do you think, please? Thanks!

Reply:
I've been talking to @dherykw and we think the best option right now is to revert our pull request. We will try to detect the bug in lwm2m-node-lib and we will provide a new commit.

Reply:
Test are back ok. NTC (with regards to this comment in this PR)